packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.ts

```diff
@@ -501,14 +506,25 @@ export default class TableColumnResizeEditing extends Plugin {
 		domEventData.preventDefault();
 		eventInfo.stop();
 
-		// The column widths are calculated upon mousedown to allow lazy applying the `columnWidths` attribute on the table.
-		const columnWidthsInPx = _calculateDomColumnWidths( modelTable, this._tableUtilsPlugin, editor );
+		this._initialMouseEventData = domEventData;
+	}
+
+	/**
+	 * Starts the resizing process after the threshold is reached.
+	 */
+	private _startResizingAfterThreshold() {
+		const domEventData = this._initialMouseEventData!;
+		const { target } = domEventData;
+
+		const modelTable = this.editor.editing.mapper.toModelElement( target.findAncestor( 'figure' )! )!;
 		const viewTable = target.findAncestor( 'table' )!;
-		const editingView = editor.editing.view;
+
+		// Calculate the initial column widths in pixels.
+		const columnWidthsInPx = _calculateDomColumnWidths( modelTable, this._tableUtilsPlugin, this.editor );
 
 		// Insert colgroup for the table that is resized for the first time.
 		if ( !Array.from( viewTable.getChildren() ).find( viewCol => viewCol.is( 'element', 'colgroup' ) ) ) {
-			editingView.change( viewWriter => {
+			this.editor.editing.view.change( viewWriter => {
```

Comment:
Personally, I'd extract the editing view to `const editingView = this.editor.editing.view;` as it was in the previous version (or at least the `editing`). It's a matter of preference, but I find it easier to review if I don't have to read the whole path to the `editing` or `view` part 🙂. Especially, when you use it more than once in the same method.